readme.md

```diff
@@ -1,6 +1,6 @@
 # Frontend practice with catalog page and hovers
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog_hovers/)
+- [DEMO LINK](https://Slepco.github.io/layout_catalog_hovers/)
```

Comment:
add this link to the description

Reply:
and try to fix it - it leads to 404. ask in the chat if you have problems with deploy